python/semantic_kernel/connectors/ai/open_ai/services/realtime/utils.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import TYPE_CHECKING, Any
+
+from openai.types.beta.realtime import (
+    ConversationItem,
+    ConversationItemCreateEvent,
+    ConversationItemDeleteEvent,
+    ConversationItemTruncateEvent,
+    InputAudioBufferAppendEvent,
+    InputAudioBufferClearEvent,
+    InputAudioBufferCommitEvent,
+    RealtimeClientEvent,
+    ResponseCancelEvent,
+    ResponseCreateEvent,
+    SessionUpdateEvent,
+)
+from openai.types.beta.realtime.response_create_event import Response
+from openai.types.beta.realtime.session_update_event import Session
+
+from semantic_kernel.connectors.ai.open_ai.services.realtime.const import SendEvents
+
+if TYPE_CHECKING:
+    from semantic_kernel.connectors.ai.function_choice_behavior import (
+        FunctionCallChoiceConfiguration,
+        FunctionChoiceType,
+    )
+    from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+    from semantic_kernel.functions.kernel_function_metadata import KernelFunctionMetadata
+
+
+def update_settings_from_function_call_configuration(
+    function_choice_configuration: "FunctionCallChoiceConfiguration",
+    settings: "PromptExecutionSettings",
+    type: "FunctionChoiceType",
+) -> None:
+    """Update the settings from a FunctionChoiceConfiguration."""
+    if (
+        function_choice_configuration.available_functions
+        and hasattr(settings, "tool_choice")
+        and hasattr(settings, "tools")
+    ):
+        settings.tool_choice = type
+        settings.tools = [
+            kernel_function_metadata_to_function_call_format(f)
+            for f in function_choice_configuration.available_functions
+        ]
+
+
+def kernel_function_metadata_to_function_call_format(
```

Comment:
Curious why this needs to live here? It looks like a duplicate of what is available in the SK's connector/ai code.